stable/artifactory-ha/OWNERS

```diff
@@ -0,0 +1,11 @@
+approvers:
+- jainishshah17
+- eldada
+- rimusz
+reviewers:
+- unguiculus
+- prydonius
+- mattfarina
+- jainishshah17
+- eldada
+- rimusz
```

Comment:
Should rimusz also be listed in the Chart.yaml as a maintainer?

Reply:
done